libmamba/tests/src/specs/test_match_spec.cpp

```diff
@@ -47,6 +47,66 @@ TEST_SUITE("specs::match_spec")
             CHECK_EQ(ms.str(), "xtensor>=0.12.3");
         }
 
+        SUBCASE("python > 3.11")
+        {
+            auto ms = MatchSpec::parse("python > 3.11").value();
+            CHECK_EQ(ms.name().str(), "python");
+            CHECK_EQ(ms.version().str(), ">3.11");
+            CHECK(ms.build_string().is_explicitly_free());
+            CHECK(ms.build_number().is_explicitly_free());
+            CHECK_EQ(ms.str(), "python>3.11");
+        }
+
+        SUBCASE("numpy < 2.0")
+        {
+            auto ms = MatchSpec::parse("numpy< 2.0").value();
```

Comment:
I mean the test should be modified as well:
```suggestion
            auto ms = MatchSpec::parse("numpy < 2.0").value();
```